### PR TITLE
[packaging] Add tf2nnpkg 20200616

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit adds tf2nnpkg.20200616 to packaging

Related: #2074
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: infra/packaging/res/tf2nnpkg.20200616 [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+# generate tflite file
+python "${ROOT}/bin/tf2tfliteV2.py" --v1 --input_path ${GRAPHDEF_FILE} \
+--output_path "${TMPDIR}/${MODEL_NAME}.tflite" \
+--input_arrays ${INPUT} --output_arrays ${OUTPUT}
+
+# convert .tflite to .circle
+"${ROOT}/bin/tflite2circle" "${TMPDIR}/${MODEL_NAME}.tflite" "${TMPDIR}/${MODEL_NAME}.circle"
+
+# optimize
+"${ROOT}/bin/circle2circle" "${TMPDIR}/${MODEL_NAME}.circle" "${TMPDIR}/${MODEL_NAME}.circle"
```
Comment (seanshpark) [inline infra/packaging/res/tf2nnpkg.20200616:87]: We are adding optimization options... what to add is a decision point, we can discuss here or in another issue.

Comment (mhs4670go) [reply infra/packaging/res/tf2nnpkg.20200616:87]: @seanshpark I'll discuss it in another issue.

**Comment by seanshpark** [comment]
@mhs4670go , you want to go as-is add add options later?

**Comment by mhs4670go** [comment]
@seanshpark Yes please:)

**Pull request merged**